doc/develop/toolchains/designware_arc_mwdt.rst

```diff
@@ -13,6 +13,9 @@ DesignWare ARC MetaWare Development Toolkit (MWDT)
       Even though ARC MWDT toolchain is used for Zephyr RTOS build, still the GNU preprocessor & GNU
       objcopy might be used for some steps like device tree preprocessing and ``.bin`` file
       generation. We used Zephyr SDK as a source of these ARC GNU tools as well.
+      To setup ARC GNU toolchain please use SDK Bundle (Full or Minimal) instead of manual installation
```

Comment:
Maybe add a reference to the Zephyr Project documentaion page which explains how to do that?

Reply:
This small note is already a part of Zephyr documentation (https://docs.zephyrproject.org/latest/develop/toolchains/designware_arc_mwdt.html). This page also contain links to both toolchains installation instructions.